kie-server-parent/kie-server-tests/kie-server-test-web-service/src/main/java/com/jboss/soap/service/acmedemo/impl/HeaderInInterceptor.java

```diff
@@ -38,26 +31,12 @@ public void handleMessage(SoapMessage message) throws Fault {
        logger.info("headerContent is {}", headerContent);
 
        //DumbEscapeHandler modifies chars over US-ASCII
-       if ("Espa&#241;a".equals(headerContent)) {
+       if ("Espa&#241;a".equals(headerContent) || "AT&T".equals(headerContent)) {
```

Comment:
As far as I understood we write `<Coupon><Number><![CDATA[AT&T]]></Number></Coupon>` as `<Coupon><Number>AT&T</Number></Coupon>`, don't we? So why do we no longer need the XPath expression to extract the coupon number? Where does this extraction happen?

Reply:
`getTextContent` now returns only the text under the xml nodes (`AT&T`), while before it was returning the whole xml stringfied: `<Coupon><Number>AT&T</Number></Coupon>`